docs/internals/security.txt

```diff
@@ -38,6 +38,30 @@ action to be taken, you may receive further followup emails.
 
 .. _our public Trac instance: https://code.djangoproject.com/query
 
+.. _reporting-guidelines:
+
+Reporting guidelines
+====================
+
+* The vulnerability must be within Django and not a third-party package or the
+  Python standard library. See the `Python security guidelines
+  <https://www.python.org/dev/security/>`_ for reporting a vulnerability in
+  Python.
```

Comment:
This is basically scoping the things we can fix, it might still be valid but needs to be reported elsewhere